images/cassandra/rootfs/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/dumb-init /bin/bash
```

Comment:
No scripts should have dumb-init. That should be container entrypoint, but never script interpreter.